central/scannerdefinitions/handler/updaterType.go

```diff
@@ -0,0 +1,39 @@
+package handler
```

Comment:
I don't think this is necessary. I think we can just do something like 

```
type updaterType int

const (
        mappingUpdaterType       updaterType = iota
        vulnerabilityUpdaterType
        v2UpdaterType
)
```

and also just pass the related key

Reply:
Updated